external-crates/move/crates/move-compiler/tests/linter/correct_public_mut_tx_context.move

```diff
@@ -0,0 +1,10 @@
+module 0x42::M {
+    use sui::tx_context::TxContext;
+    public fun mint(_ctx: &mut TxContext) {
+        
+    }
+}
+
+module sui::tx_context {
+    struct TxContext has drop {}
+}
```

Comment:
Please configure your editor to have a trailing new line

Reply:
refactored